tests/regression/fn/fn_vertical_indirection.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2021, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <cstddef>
+#include <gtest/gtest.h>
+
+#include <gridtools/fn/cartesian.hpp>
+#include <gridtools/fn/unstructured.hpp>
+
+#include <fn_select.hpp>
+#include <test_environment.hpp>
+
+namespace {
+    using namespace gridtools;
+    using namespace fn;
+    using namespace literals;
+
+    struct derivative_stencil {
+        auto operator()() const {
+            return [](auto const &field, auto const &offset) {
+                const auto shifted = shift(field, unstructured::dim::vertical{}, deref(offset));
+                return deref(field) - deref(shifted);
+            };
+        }
+    };
+
+    constexpr auto parabolic = []([[maybe_unused]] auto horizontal_idx, auto vertical_idx) -> float {
```

Comment:
```suggestion
    constexpr auto parabolic = [](auto /*vertex*/, auto k) -> float {
```
For consistency with other GT code: use commented name instead of `[[maybe_unused]]`, same index names as in `fn_unstructured_nabla.cpp`.

Reply:
Replaced with conventional notation.